test/paper-button-behavior.html

```diff
@@ -73,7 +75,8 @@
 
       test('receives focused', function() {
         MockInteractions.focus(button);
-        assert.equal(button._elevation, 3);
+        assert.equal(button.elevation, 3);
+        assert.ok(button._ripple);
```

Comment:
This is a minor hygiene note, but it might be nice to make a public API to measure the state of the ripple being present, rather than measuring protected implementation externally this way.